README.md

```diff
@@ -139,3 +139,5 @@ Using Shell:
 ```
 
 The output will be located in the folder build\renders.
+
+## Test
```

Comment:
Why are you forcing a rebuild on this? There is no JIRA or description on the PR